doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,9 +63,385 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+| STORAGE_CLUSTER_BACKEND_IP | List | Yes |   |
+| STORAGE_CLUSTER_BACKEND_GW | List | Yes |   |
+| BACKUP_POLICY | List | Yes |   |
+
+<p align="center"><b>Table 6-3-13-2:</b> Distribution storage pool info.</p>
+
+
+<a name="6.3.14"></a>
+### 6.3.14 Software integration information
+After VIM and Storage software installation finished, parameters willl be needed in integration process of VIM and Storage,
+the parameters should be defined in advance.
+
+#### 6.3.14.1 VIM Context
+Parameters from VIM vendor for integration.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| VENDOR | String | Yes |  |
+| AUTHORIZATION | String | Yes | One-way or Two-way authentication |
+| JOINT_WAY | String | Yes | by ISCSI or client |
+| VIM_CERTIFICATES_PATH | String | Yes | Full path for certificates that used for integration |
+
+<p align="center"><b>Table 6-3-14-1:</b> VIM context Information.</p>
+
+#### 6.3.14.2 Storage Context
+Parameters from storage vendor for integration.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| VENDOR | String | Yes |  |
+| AUTHORIZATION | String | Yes | One-way or Two-way authentication |
+| JOINT_WAY | String | Yes | by ISCSI or client |
+| DRIVER_FULL_NAME | String | Yes | Full path for storage driver |
+| CEPH_CONFIG_PATH | String | Yes | Full path for ceph.conf storage |
+| IS_PIM_JOINT | String | Yes | whether integrate with PIM, usaually "YES" for this value |
+| STORAGE_SOFTWARE_VERSION | String | Yes |  |
+
+<p align="center"><b>Table 6-3-14-2:</b> Storage context Information.</p>
+
+#### 6.3.14.3 Storage Client context
+This table defines the parameters for integration with storage client
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| JOINT_WAY | String | Yes | integration method for storage client, for example, RBD  |
+| COMPONENT_TYPE | String | Yes | for example: cinder, glance or nova |
+| AUTHORIZATION_USER | String | Yes | match with the component type |
+| KEYRING_FILENAME | String | Yes | Full path for keyring file, this should match the authentication user, |
+
+<p align="center"><b>Table 6-3-14-3:</b> Storage client context.</p>
+
+<a name="6.3.15"></a>
+### 6.3.15 Device Management information
+
+#### 6.3.15.1 SERVER PIM ACCOUNT
+Servers are managed by redfish, credentials should be the same for same type of device  
```

Comment:
How is this section different from the BMC information above, seems redundant.

Reply:
Make sense, will discuss with people who use it, if it's OK to remove